src/apps/applets/api/applets.py

```diff
@@ -1,52 +1,63 @@
 from fastapi import Body, Depends
 
+import apps.applets.domain as domain
 from apps.applets.crud.applets import AppletsCRUD
 from apps.applets.crud.roles import UserAppletAccessCRUD
-from apps.applets.domain.applets import (
-    Applet,
-    AppletCreate,
-    PublicApplet,
-    UserAppletAccessCreate,
-)
 from apps.applets.domain.constants import Role
 from apps.authentication.deps import get_current_user
 from apps.shared.domain.response import Response, ResponseMulti
 from apps.shared.errors import NotContentError
 from apps.users.domain import User
 
 
+# TODO: Add logic to allow to create applets by permissions
+# TODO: Restrict by admin
 async def create_applet(
     user: User = Depends(get_current_user),
-    schema: AppletCreate = Body(...),
-) -> Response[PublicApplet]:
-    applet: Applet = await AppletsCRUD().save(schema=schema)
+    schema: domain.applet_create.AppletCreate = Body(...),
+) -> Response[domain.applet.Applet]:
```

Comment:
Why do we need this extra encapsulation?

Reply:
We use a great divided structure and all our files are isolated. The namespace also provides the ability not to think about module-based imports.
I don't understand the actual reason to do that... It is much harder to read. Also, it is not consistent with the rest of the code. I think if we'd like to implement it in that way we have to create additional PR with the previous alignment in a team. Because what is the reason not to do that for PublicApplet for example?

Reply:
faced issues with black --check . && isort --check ., where it says imports order is not correct. I have tried several ways, only this way worked